src/Training.cpp

```diff
@@ -200,17 +200,14 @@ void Training::dump_stats(const std::string& filename) {
 }
 
 void Training::dump_stats(OutputChunker& outchunk) {
-    {
-        std::stringstream out;
-        out << "1" << std::endl; // File format version 1
-        outchunk.append(out.str());
-    }
+    std::stringstream out;
+    out << "1" << std::endl; // File format version 1
     for (const auto& step : m_data) {
         std::stringstream out;
         out << step.net_winrate
             << " " << step.root_uct_winrate
             << " " << step.child_uct_winrate
             << " " << step.bestmove_visits << std::endl;
```

Comment:
Hm..  I think this drops all the stuff inside this loop.

Reply:
Good catch, fixed. I don't think this function is used, in LZ we use it to analyze the resign rate, there is a special GTP command to call this function.